### PR TITLE
add entries to troubleshooting

### PR DESCRIPTION
**Pull request by zr-msft**
incorporates #125

### CODE REVIEW

File: content/en/docs/troubleshooting.md [hunk 1]
```diff
@@ -69,3 +74,88 @@ IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
...
+
+This error likely is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}}
+
+## When I verify an artifact, I get the error '"$HOME/.config/notation/truststore/x509/ca/mytruststore" does not exist'
+
+This error indicates the trust store doesn't exist or the trust store name is not correct. Trust store typically contains a set of certificate files, where the trust identities are retrieved to verify signatures. You can use `notation cert add` to add trust stores.
+
+The above error shows the type of trust store is `ca`, and the trust store name is `mytruststore`. 
+
+To verify `ca` and `mytruststore` exist, use `notation cert list`, then confirm whether the type of store `ca` and store name `mytruststore` are in the list with the right certificate file stored.
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:99]: Per Toddy's suggestion, maybe we can show users how to fix it by using command `notation cert add --type ca --store mytruststore mycertfile.crt`

File: content/en/docs/troubleshooting.md [hunk 2]
```diff
@@ -69,3 +74,88 @@ IMAGE@sha256:111222333444555666777888999000aaaabbbbcccddddeeeeffff00011122233
...
+
+When running a notation CLI command, use the `--verbose` flag to enable verbose logging. For example:
+
+```shell
+notation verify --verbose localhost:5000/net-monitor@sha256:sha256:xxx
+```
+
+## When I verify an artifact, I get the error 'Error: open $HOME/.config/notation/trustpolicy.json: no such file or directory'
+
+This error likely is likely related to trust policy configuration. Verify you have a trust policy set up before you attempt to verify an artifact. For more details, see [Manage trust policies]({{< ref "/docs/how-to/manage-trust-policy" >}}
```
Comment (yizha1) [inline content/en/docs/troubleshooting.md:91]: nit: two `likely`

**Review by yizha1 — COMMENTED** [review]
Thanks @zr-msft. A few comments

File: content/en/docs/troubleshooting.md (hunk 1 above)
Comment (zr-msft) [reply content/en/docs/troubleshooting.md:101]: good call, fixed

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**